indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/PendingTaskBasedWorkerProvisioningStrategy.java

```diff
@@ -201,27 +235,33 @@ private int getScaleUpNodeCount(
     {
       final int minWorkerCount = workerConfig.getAutoScaler().getMinNumWorkers();
       final int maxWorkerCount = workerConfig.getAutoScaler().getMaxNumWorkers();
-      final Predicate<ImmutableWorkerInfo> isValidWorker = ProvisioningUtil.createValidWorkerPredicate(config);
-      final int currValidWorkers = Collections2.filter(workers, isValidWorker).size();
+      log.info("Min/max workers: %d/%d", minWorkerCount, maxWorkerCount);
```

Comment:
It's better to log that during config update.

Reply:
I think it also makes sense in the context of surrounding log messages rather than detached, so left